README.md

```diff
@@ -67,30 +67,42 @@ Parameter | Description | Default
 `base.repository` | LocalEGA container image repository | `nbisweden/ega-base`
 `base.imageTag` | LocalEGA container image tag | `latest`
 `base.imagePullPolicy` | LocalEGA container image pull policy | `IfNotPresent`
-`config.deploy` | If true, deploy ConfigMaps | `true`
 `config.broker_connection_attempts` | Connection attempts before timing out | `30`
+`config.broker_enable_ssl` | Use SSl for broker conneciton | `false`
+`config.broker_heartbeat` | Heartbeat timeout balue | `0`
+`config.broker_host` | hostname of local MQ broker | `""`
 `config.broker_retry_delay` | time in seconds between each connection attempt | `10`
-`config.broker_username` | username for accessing local MQ service | `""`
-`config.cega_endpoint` | URL to CEGA user service | `""`
-`config.cega_endpoint_json` | Prefix in the JSON response that contains the user info| `response.result`
+`config.broker_username` | username for accessing local MQ service | `guest`
+`config.cega_host` | hostname of CEGA users service | `""`
+`config.cega_endpoint` | URL to CEGA user service | `"/lega/v1/legas/users/%s?idType=username"`
+`config.cega_endpoint_json` | Prefix in the JSON response that contains the user info | `response.result`
 `config.cega_mq_host` | CEGA MQ hostname | `""`
 `config.cega_vhost` | CEGA MQ vhost path | `""`
-`config.cega_port` | CEGA MQ acces port | `5271`
+`config.cega_port` | CEGA MQ acces port | `5272`
```

Comment:
I think this is the default port when connecting to CRG ? but it should be `5672` i assume most people will use the default RabbitMQ - but not a hard requirement

In the `values.yml` i see it is `5672`